catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+# Install dependencies
+if [ -f /usr/bin/pacman ] 
+then
+  sudo pacman -S curl wget unzip git
+elif [ -f /usr/bin/emerge ]
+then
+  sudo emerge -av curl wget unzip git
+elif [ -f /usr/bin/apt ]
+then
+   sudo apt install curl wget unzip git
+elif [ -d /nix ]
+then
+  sudo nix-shell -p curl wget unzip git
+elif [ -f /usr/bin/dnf ]
+then
+  sudo dnf install curl wget unzip git
+fi
+
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Get the latest release of SFP and unzip it
+mkdir SFP 
+cd SFP || exit 1
+curl -s https://api.github.com/repos/PhantomGamers/SFP/releases/latest \
+| grep "SFP_UI-SelfContained-linux-x64.zip" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+unzip *.zip
+chmod +x SFP_UI
+./SFP_UI 
+
+# Install metro
+if [ -d ~/.steam ]
+then
+  mkdir -p ~/.steam/steam/skins/
+  git clone https://github.com/minischetti/metro-for-steam ~/.steam/steam/skins/catppuccin
+  export METRO="$HOME/.steam/steam/skins/catppuccin"
+elif [ -d ~/.var/app/com.valvesoftware.Steam ]
+then
+  mkdir -p ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins
+  git clone https://github.com/minischetti/metro-for-steam ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin
+  export METRO="$HOME/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin"
+fi
+ 
+# Install the metro patch
+cd ../ 
+mkdir UPMetro
+cd UPMetro || exit 1
+git clone https://github.com/redsigma/UPMetroSkin
+cp -r 'Unofficial 4.x patch'/'Main Files'/* "$METRO"
```

Comment:
```suggestion
cp -r 'Unofficial 4.x patch'/'Main Files'/* "$install_path"
```
just the new variable name from the previous suggestion